### ISSUE TITLE
No output

### ISSUE THREAD

**Issue by DaveyD** [dfinke/vscode-pandoc#9]
Hi, I just installed Pandoc and your pandoc extension for vs-code
After installing, I opened a markdown file and tested all 3 formats
I wasnt able to get any of them to work!

Is there a setting that I am missing?

Please let me know
Thanks,
David

**Comment by BenjaminAbt** [comment]
Same here. Did you fixed it?

**Comment by dfinke** [comment]
If you run `pandoc` against your md file at the command line, do you get errors?

`pandoc test.md -o test.pdf`

**Comment by dfinke** [comment]
Thanks for trying. Could you post the md and the version of the ext and vs code you're running.

**Comment by BenjaminAbt** [comment]
I could reproduce and fix it to the same time.

I've installed this extension yesterday and also installed pandoc afterwards (without Latex, because I do not need PDF here). After the installation I instantly tested the stuff via extension and it failed. CLI worked fine.

Now, back at home the extension works. Weird..

Well, I have now installed this extension and pandoc on my Laptop and: it does not work. CLI works here too. Then - just to have the same workflow - I've rebooted the laptop and here we go: it works.

It seems that there is at any point a restart required.

**Comment by DaveyD** [comment]
@dfinke - Good news this time! :) 
I just updated my VScode and retried your exension - This time it seems to work great for HTML and DOCX.
However, for PDF it does **not** work.

Running from the command line returns the following error:
`pandoc: pdflatex not found. pdflatex is needed for pdf output.`

I am not sure what that is - is that an additional resource necessary to run?

Thanks,
David

**Comment by BenjaminAbt** [comment]
Dave, see pandoc installation note: you need LaTeX for PDF export.

**Comment by DaveyD** [comment]
Ok, got it - never went over pandoc docs
Thanks for that pointing that out Benjamin

**Comment by divillysausages** [comment]
Just to chime in here with some possible causes

- Running the task in the command line, I had to install a number of additional packages (Windows). I'd originally set pandoc to ask me beforehand, so this might be blocking the case
- I don't have Adobe Reader installed - I just use the web browser for looking at pdfs. Both the docx and html versions opened their respective editors, so it might be that

**Comment by po5i** [comment]
Yeah, I'm working on a total different project and just realized that this `tex2pdf.XXXX` tries to be created on the directory where `pandoc` was executed.

**Comment by furicle** [comment]
This is related to #21 

I just tested and discovered if I install the plug in as administrator, it works fine.  But as a standard user, it can't find pandoc.

It must be looking at the wrong path.


Note I have a system wide VS Code install

**Comment by furicle** [comment]
Hmmm - but the path is correct for both users....
